hg-fast-export.py

```diff
@@ -135,6 +169,21 @@ def export_file_contents(ctx,manifest,files,hgtags,encoding=''):
       filename=file.decode(encoding).encode('utf8')
     else:
       filename=file
+    if filename[:5] == ".hglf":
+      sys.stderr.write("Detected large file")
+      filename = filename[5:]
+      #should detect where the large files are located
+      lfsFileCached = lfutil.findfile(ctx.repo(), d.strip('\n'))
+      if lfsFileCached is not None:
+        d = generate_git_lfs_pointer(lfsFileCached)
+      else:
+        # Autodownloading from the mercurial repository would be an issue as there is a good chance that we may
+        # need to input some username and password. This will surely break the fast-export as there will be
+        # some unexpected output.
+        sys.stderr.write("Large file wasn't found in local cache")
+        sys.stderr.write("Please clone with --all-largefiles")
+        sys.stderr.write("Or pull with --lfrev %s" % (str(ctx.rev())))
+        sys.exit(3) # closing in the middle of import will revert everything to the last checkpoint
```

Comment:
Have you tested this? There are no line-feeds and no spaces between the printouts and "Or" should not be capitalized.